english/thunder-carrots/thunder-carrots.md

```diff
@@ -0,0 +1,11 @@
+You dream of walking to the microwave in the kitchen.
+And you warm up a little soup. But then something happens.
+
+Out from the microwave, a dozen exploding thunder carrots swarm you smothering you as you cower.
+You fall to the ground and hit your head. You come to, and,
+
+Yes!
+
+That's how to [clone](https://help.github.com/articles/fork-a-repo/) a remote!
```

Comment:
This link seems to be an absolute link to GitHub rather than a relative link.  If you are linking to another page in the create-your-own-adventure repository, please use relative links to do so.  See the README of this repository for more information.